utils/test/src/main/kotlin/Utils.kt

```diff
@@ -80,16 +80,25 @@ fun patchExpectedResult(
         .replaceIfNotNull("<REPLACE_URL_PROCESSED>", urlProcessed)
 }
 
+class ReplacementsBuilder {
+    val replacements = mutableMapOf<String, String>()
+
+    infix fun String.replaces(value: String) {
+        replacements[value] = this
+    }
+}
+
 fun patchExpectedResult2(
     expectedResultFile: File,
     definitionFile: File,
-    custom: Map<String, String> = emptyMap()
+    block: ReplacementsBuilder.() -> Unit = {}
 ): String {
     val projectDir = definitionFile.parentFile
     val vcsDir = VersionControlSystem.forDirectory(projectDir)!!
     val vcsUrl = vcsDir.getRemoteUrl()
     val vcsRevision = vcsDir.getRevision()
     val vcsPath = vcsDir.getPathToRoot(projectDir)
+    val custom = ReplacementsBuilder().apply(block).replacements
```

Comment:
... change this to `val custom = buildMap(block)` (or even inline it below).

Reply:
But actually, when getting rid of `<REPLACE_GITHUB_PROJECT>`, there's maybe no reason anymore to turn custom replacements into a lambda at all.

Reply:
> But actually, when getting rid of `<REPLACE_GITHUB_PROJECT>`, there's maybe no reason anymore to turn custom replacements into a lambda at all.

You like readability without lambda more?

Reply:
> You like readability without lambda more?

It does not make much of a difference to me. Feel free to keep it at a lamda, if you like. Maybe, yes, it's a tad nicer after all.